models/models.go

```diff
@@ -154,7 +154,15 @@ func (codelist *CodeList) validateCodelistHRef(ctx context.Context) bool {
 		log.Error(ctx, "error parsing codelist.href", err)
 		return false
 	}
-	urlPathBool := strings.Contains(hRefURL.Path, "/code-lists") && strings.Contains(hRefURL.Path, codelist.ID)
+
+	urlPathBool := strings.Contains(
+		strings.ToLower(hRefURL.Path),
+		strings.ToLower("/code-lists"),
+	) && strings.Contains(
+		strings.ToLower(hRefURL.Path),
+		strings.ToLower(codelist.ID),
+	)
+
```

Comment:
Are we affecting how CMD works if we change this here?

Reply:
We're just allowing the URL to include a different case for the id portion from the actual provided id. It won't actually modify anything provided and nothing existing will change. Also AFAIK the URLs are only meant for human-reading.